readyplayerme/rpm-unreal-sdk
# [SDK-372] Automated Cache Management for Unreal

## [SDK-372](https://ready-player-me.atlassian.net/browse/SDK-372)

## Description

-   Added AvatarManifest for saving the avatars load time
-   Changed AvatarCacheHandler to support the manifest

## Changes

#### Added

-   Added AvatarManifest for saving the avatars load time

#### Updated

-   Changed AvatarCacheHandler to support the manifest

## How to Test

-   Change the AvatarLimit in the project settings and check if the old avatars are removed when the new ones are added

## Checklist



[SDK-372]: https://ready-player-me.atlassian.net/browse/SDK-372?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
